kebechet/managers/update/update.py

```diff
@@ -710,26 +711,14 @@ def run(self, labels: list = []) -> Optional[dict]:
             runtime_environment_names = [
                 e["name"] for e in thoth_config.list_runtime_environments()
             ]
-
             overlays_dir = thoth_config.content.get("overlays_dir")
 
-            runtime_environments: List[Optional[str]]
-            if self.runtime_environment:
-                if self.runtime_environment not in runtime_environment_names:
+            results: dict = {}
+            for e in self.runtime_environments or []:
+                if e not in runtime_environment_names:
                     # This is not a warning as it is expected when users remove and change runtime_environments
                     _LOGGER.info("Requested runtime does not exist in target repo.")
-                    return None
-                runtime_environments = [self.runtime_environment]
-            else:
-                if overlays_dir:
-                    runtime_environments = runtime_environment_names
-                elif runtime_environment_names:
-                    runtime_environments = [runtime_environment_names[0]]
-                else:
-                    runtime_environments = [None]
```

Comment:
I think we lost the "default" functionality here.  I think the for loop should probably be default `[None]` instead of `[]`